src/main/java/org/hisp/dhis/tasks/CreateTrackedEntityAttributeTask.java

```diff
@@ -0,0 +1,72 @@
+package org.hisp.dhis.tasks;
+
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.hisp.dhis.cache.GeneratedTrackedEntityAttribute;
+
+import java.util.UUID;
+
+import static io.restassured.RestAssured.given;
+
+public class CreateTrackedEntityAttributeTask
```

Comment:
I'm wondering if there's a use case where several tracked entity attributes are posted in bulk? If so, could be good to allow passing a number of TEA like we do in TrackedEntityInstances test.

Reply:
Maybe that would be good to make the test setup faster, but then it will introduce the problem to make the TEA available for different users.

This kind of sharing between users is something that we will need to tackle, now for simplicity, any task is responsible for its own TEA

Reply:
Why would it introduce a sharing problem? I mean, sending several TEA in the same payloads.